jetty-io/src/test/java/org/eclipse/jetty/io/SelectorManagerTest.java

```diff
@@ -120,39 +114,36 @@ protected void connectionFailed(SelectableChannel channel, Throwable ex, Object
         {
             SocketChannel client1 = SocketChannel.open();
             client1.configureBlocking(false);
-            client1.connect(address);
-            timeoutConnection.set(connectTimeout * 110 / 100);
-            final CountDownLatch latch1 = new CountDownLatch(1);
+            assertFalse(client1.connect(address));
             selectorManager.connect(client1, new Callback()
             {
                 @Override
                 public void failed(Throwable x)
                 {
-                    latch1.countDown();
+                    failedConnectionLatch.countDown();
                 }
             });
-            assertTrue(latch1.await(connectTimeout * 3, TimeUnit.MILLISECONDS));
+            assertTrue(failedConnectionLatch.await(connectTimeout * 2, TimeUnit.MILLISECONDS));
```

Comment:
`failedConnectionLatch` has count=1, but it is awaited twice, which does not seem correct.

Reply:
My bad, the code is fine.